espnet_model_zoo/table.csv

```diff
@@ -98,3 +98,4 @@ librispeech,asr,kamo-naoyuki/librispeech_asr_train_asr_conformer5_raw_bpe5000_sc
 librispeech,asr,kamo-naoyuki/librispeech_asr_train_asr_conformer5_raw_bpe5000_frontend_confn_fft400_frontend_confhop_length160_scheduler_confwarmup_steps25000_batch_bins140000000_optim_conflr0.0015_initnone_sp_valid.acc.ave,https://zenodo.org/record/4543003/files/asr_train_asr_conformer5_raw_bpe5000_frontend_confn_fft400_frontend_confhop_length160_scheduler_confwarmup_steps25000_batch_bins140000000_optim_conflr0.0015_initnone_sp_valid.acc.ave.zip?download=1,16000,en,,1.7.1,0.9.6,8eff1a9,true
 librispeech,asr,kamo-naoyuki/librispeech_asr_train_asr_conformer5_raw_bpe5000_frontend_confn_fft512_frontend_confhop_length256_scheduler_confwarmup_steps25000_batch_bins140000000_optim_conflr0.0015_initnone_sp_valid.acc.ave,https://zenodo.org/record/4543018/files/asr_train_asr_conformer5_raw_bpe5000_frontend_confn_fft512_frontend_confhop_length256_scheduler_confwarmup_steps25000_batch_bins140000000_optim_conflr0.0015_initnone_sp_valid.acc.ave.zip?download=1,16000,en,,1.7.1,0.9.6,8eff1a9,true
 laborotv,asr,Shinji Watanabe/laborotv_asr_train_asr_conformer2_latest33_raw_char_sp_valid.acc.ave,https://zenodo.org/record/4304245/files/asr_train_asr_conformer2_latest33_raw_char_sp_valid.acc.ave.zip?download=1,16000,jp,,1.4.0,0.9.5,c40d6e2,true
+mls,asr,ftshijt/mls_asr_transformer_valid.acc.best,https://zenodo.org/record/4458452#.YDUeb89KhBw.16000,es,,1.6.0,0.9.2,c0c3724fe660abd205dbca9c9bbdffed1d2c79db,true
```

Comment:
Please write the url of a zip file instead and change the commit id to 7char str.

Now you are admin of this repo, so you can merge if ci is passing.

Reply:
OK, thanks!